perl/p5-tcl-ptk/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           perl5 1.0
+
+perl5.branches      5.26
+perl5.setup         Tcl-pTk 0.92
+
+platforms           darwin
+maintainers         {@chrstphrchvz gmx.us:chrischavez} openmaintainer
+license             {Artistic-1 GPL}
+
+supported_archs     noarch
+
+description         Tcl::pTk - Interface to Tcl/Tk with Perl/Tk compatible sytax
+
+long_description    Tcl::pTk interfaces perl to an existing Tcl/Tk \
+                    installation on your computer. It has fully perl/tk \
+                    compatible syntax for running existing perl/tk scripts, \
+                    as well as direct-tcl syntax for using any other Tcl/Tk \
+                    features. \
+                    \n\nUsing this module an interpreter object is created, \
```

Comment:
I'm not sure about the \n syntax. That said, maybe it works. Someone might know.

Reply:
It's not hugely important, but it's one of only two ways I've found for preserving line breaks in the description (the other is putting the whole description in quotes). I haven't found many other Portfiles that use this; e.g. HexFiend and nfft-3 use it for bullets.

Reply:
But it does indeed work?